geofield.formatters.inc

```diff
@@ -213,6 +213,8 @@ function geofield_field_formatter_settings_summary($field, $instance, $view_mode
  * Implements hook_field_formatter_view().
  */
 function geofield_field_formatter_view($entity_type, $entity, $field, $instance, $langcode, $items, $display) {
+  module_load_include('inc', 'geofield', 'vendor/phayes/geophp/geoPHP');
```

Comment:
we probably don't need `vendor/phayes` here. It's very unlikely that one module will ever include more than one library, or have more than one vendor.